src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py

```diff
@@ -201,7 +132,7 @@ def _updated_timestamps_to_offset_time(
         return new_values
 
     @staticmethod
-    def _get_x_width_list(x_values: list[float]) -> list[float]:
+    def get_x_width_list(x_values: list[float]) -> list[float]:
```

Comment:
Move this function to the StackedBarSource class.

Reply:
Fixed it in [7413d0f](https://github.com/tier4/CARET_analyze/pull/307/commits/7413d0f72a3ce8e0052c712da4b7db406081e5be).